fast_pred/src/csrmatrix.cpp

```diff
@@ -24,6 +24,6 @@ std::string CSRMatrix::toString() {
         "indptr:" + indptr.toString() +
         "data:" + data.toString() +
         "rows:" + lm_utils::intToString(this->rows) +
-        "cols:" + lm_utils::intToString(this->cols);
+        " cols:" + lm_utils::intToString(this->cols);
```

Comment:
is this a mistake?

Reply:
Previously, the output was `rows:ncols:n`

Reply:
So I assume the previous toStrings are adding some whitespace